### DISCUSSION TITLE
Oops… The properties sheet could not be loaded: Failed to fetch

### DISCUSSION THREAD

**Discussion by wuzhuxizhen** [FlowmapBlue/FlowmapBlue#67] [Q&A] [answered]
Hi, what's the mean of ”Oops… The properties sheet could not be loaded: Failed to fetch“ problem with.

I also try to used another test with less rows, the result is ok.

Thanks in advance.

**Comment by ilyabo** [comment, marked as answer]
Since the app is complaining about the properties sheet, this shouldn't have to do with the number of rows. Either there is no sheet named "properties" in your spreadsheet or there's some other issue with it. I could take a look if you could share the URL of your map which is failing.

**Reply by wuzhuxizhen** [reply]
Hi ilyabo, you'are right. I didn't share the URL before. After I click the share button, it has been done.  Thanks a lot.)
